.gitignore

```
@@ -14,17 +14,12 @@ package-lock.json
 *.swp
 *.swo
 
-
 packages/**/dist/
-examples/**/build/
-examples/**/public/i18n/
-
-#sandbox:
-examples/**/static/dist/
-examples/**/static/manifest.json
-examples/**/webpack-assets.json
-examples/**/webpack-stats.json
 
+examples/**/build/
 examples/**/config/*
 !examples/**/config/custom-environment-variables.*
 !examples/**/config/default.*
+!examples/**/config/development.*
```

```suggestion
!examples/**/config/development.*
!examples/**/config/production.*
```

This file is tracked so it probably shouldn't be ignored?

---

`production` files may contain some very sensitive information, so I wouldn't keep it in the GIT simply to avoid such cases. Besides, this is a template app, if a developer/agency really needs this - they could change it any time.

---

current we do have this file in the git as a template file, but any changes to it are being ignored

---

As far as I know, when a file is tracked, git detects changes even though it is ignored. Indeed, when I change `shop-with-blog/client/config/production.json`, it shows up in `git status`.

I am not aware of a way to create such a template file. There is `update-index --assume-unchanged` but I believe that only works locally.

Unless you know a way to ignore changes while keeping it tracked, I suggest we remove the `production.json` file and keep it ignored.

---

@nimobeeren heh, you're right, something went wrong :) but still, I wouldn't encourage people to keep it in git (at least on the `client/.gitignore` level)

---

@Borales If you agree, could you remove `production.json` from git?

---

@nimobeeren  you are right, I think I need to remove `production.json` from repo